table/builder.go

```diff
@@ -109,6 +111,16 @@ func (b Builder) keyDiff(newKey []byte) []byte {
 }
 
 func (b *Builder) addHelper(key []byte, v y.ValueStruct) {
+	// Block start
+	if b.counter == 0 {
```

Comment:
I don't think it will make sense. `finishBlock` is called after the block is completed. We want to add the `key-offset` pair before the block starts.

Reply:
It will easy to add this `finishBlock` also. 
`
ko := &pb.KeyOffset{
			Key:    y.Copy(b.baseKey),
			Offset: uint32(b.baseOffset),
			Len:    uint32(b.buf.Len()) - b.baseOffset,
		}
`
I think it will be cleaner to write there.

Reply:
Done.